becquerel/core/rebin.py

```diff
@@ -64,33 +64,40 @@ def _linear_offset(slope, cts, low, high):
 @nb.jit(nb.f8(nb.f8, nb.f8, nb.f8), nopython=True)
 def _slope_integral(x, m, b):
     '''
-    Calculate the integral of our quadratic given some slope and offset.
+    The indefinite integral of y = mx + b, with an x value substituted in.
 
     Args:
-      x:
-      m:
-      b:
+      x: the x-value
+      m: the value of the slope
+      b: the y-offset value
+
+    Returns:
+      The indefinite integral of y = mx + b, with an x value substituted in.
+      (m x^2 / 2 + b x)
     '''
     return m * x**2 / 2 + b * x
 
 
 @nb.jit(nb.f8(nb.f8, nb.f8, nb.f8, nb.f8, nb.f8), nopython=True)
-def _counts(slope, offset, cts, low, high):
+def _counts(m, b, x_low, x_high):
```

Comment:
@chunhochow travis failed because of the incorrect number of args in `_counts` compared to the type def in the `jit` decorator. Can you update?